index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="profile">
+        <img class="pic" src="./images/TimScott.jpg" alt="Tim Scott" />
+        <div class="name">Tim Scott</div>
+        <div class="title">Senate (R-SC 1st District) - 2014</div>
+        <div class="separator"></div>
+        <div class="funds">
+          <div class="number">$82,967</div>
+          <div class="type">Raised</div>
+        </div>
+        <div class="donors">
+          <div class="number">9,203</div>
+          <div class="type">Donors</div>
+        </div>
+      </div>
+      <div class="profile">
+        <img class="pic" src="./images/MarkSanford.jpg" alt="Mark Sanford" />
+        <div class="name">Mark Sanford</div>
+        <div class="title">House  (R-SC 1st District) - 2014</div>
+        <div class="separator"></div>
```

Comment:
same as above--to reiterate, the reason I bring this up is so that you don't have empty DOM elements sitting around.